src/main/java/org/isf/sms/dto/SmsDTO.java

```diff
@@ -41,25 +41,25 @@ public class SmsDTO {
 	private LocalDateTime smsDateSched;
 	
 	@NotNull
-	@Schema(description = "SMS target phone number", example="+237671302313")
+	@Schema(description = "SMS target phone number", example="+237671302313", maxLength = 45)
```

Comment:
All other phone numbers are 50 characters long.

Reply:
![sms](https://github.com/informatici/openhospital-api/assets/101590821/f5581d15-05ad-466b-a915-4f3fc512b649)
all those size are take in the database.